File: Master/Year 1/Semester 1/AMSS/Proiect/app/src/main/java/ro/robertto/rescuepets/presentation/activities/GuidePage2.java
package ro.robertto.rescuepets.presentation.activities;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import ro.robertto.rescuepets.databinding.SecondPageGuideBinding;

public class GuidePage2 extends AppCompatActivity {

    private SecondPageGuideBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = SecondPageGuideBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.ThirdPageButton.setOnClickListener(view -> {
            Intent intent = new Intent(this, GuidePage3.class);
            startActivity(intent);
        });

        binding.ThirdPageButtonSkip.setOnClickListener(view -> {
            Intent intent = new Intent(this, LogInActivity.class);
            startActivity(intent);
        });
    }
}

File: Anul 3/Semestrul 2/Android/Proiect/EmergencyApp/app/src/main/java/com/lifeSavers/emergencyapp/activities/GuidePage2.kt
package com.lifeSavers.emergencyapp.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.lifeSavers.emergencyapp.databinding.SecondPageGuideBinding

class GuidePage2 : AppCompatActivity() {

    private lateinit var binding: SecondPageGuideBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = SecondPageGuideBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.ThirdPageButton.setOnClickListener {
            val intent = Intent(this, GuidePage3::class.java)
            startActivity(intent)
        }

        binding.ThirdPageButtonSkip.setOnClickListener {
            val intent = Intent(this, LogInActivity::class.java)
            startActivity(intent)
        }
    }
}
